README.md

```diff
@@ -5,6 +5,10 @@ Python inferface to MUMPS ([MUltifrontal Massively Parallel sparse direct Solver
 
 It supports all four types (single real, double real, single complex and double complex).
 
+## Caveats
+
+For now, it only works with the **SEQUENTIAL** version of MUMPS.
```

Comment:
Sorry, this is really a detail, but it would be much clearer to use `MUMPS.py` or "this package" instead of "it".